routers/repo/view.go

```diff
@@ -210,9 +211,26 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 			}
 
 			var output bytes.Buffer
-			lines := strings.Split(fileContent, "\n")
+			var terminator string
```

Comment:
By using a single terminator, we aren't able to correctly render files that sometime use `\r\n` and in other places use `\n`? Is there a reason we can't just do

```
lines := strings.Split(fileContent, "\n")
for index, line := range lines {
	if index < len(lines) - 1 {
		line += "\n"
	}
	output.WriteString(fmt.Sprintf(`<li class="L%d" rel="L%d">%s</li>`, index+1, index+1, line))
}
```

Since both `\r\n` and `\n` end in `\n`, it seems to me this would work, and it would handle the case where a file mixes `\r\n` and `\n`